packages/toolpad-app/src/toolpad/AppEditor/PageEditor/RenderPanel/NodeHud.tsx

```diff
@@ -167,6 +169,7 @@ export default function NodeHud({
   onDuplicate,
   isOutlineVisible = false,
   isHoverable = true,
+  isHovered = false,
```

Comment:
Since the dashed border doesn't really just mean the element is "hovered" anymore we might want to give this prop a different name just to avoid confusion. Maybe `isPreviewed`? Not sure, also not a super big deal.

Reply:
Ok, I think it's fine to keep it.